spec/function.dd

```diff
@@ -2957,14 +2957,19 @@ $(H2 $(LNAME2 main, $(D main()) Function))
         It gets called after all the module initializers are run, and
         after any unittests are run.
         After it returns, all the module destructors are run.
-        $(D main()) must be declared using one of the following forms:
+        $(D main()) must be declared as follows:
```

Comment:
Only mentioning the function signature is somehwat misleading w.r.t. `auto` because further restrictions apply to the inferred return type (must be `void`, `int` or `noreturn`). It would be good to be explicit and mention it in an additional paragraph below the grammar.

Reply:
I've documented that now.

Reply:
Thanks